data/Pandora.Definitions.MicrosoftGraph.Beta/CommonTypes/Constant-X509CertificateIssuerHintsConfigurationState.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+using Pandora.Definitions.Attributes;
+using System.ComponentModel;
+
+namespace Pandora.Definitions.MicrosoftGraph.Beta.CommonTypes;
+
+[ConstantType(ConstantTypeAttribute.ConstantType.String)]
+internal enum X509CertificateIssuerHintsConfigurationStateConstant
+{
+    [Description("Disabled")]
+    @disabled,
```

Comment:
@manicminer these two likely need to be around the other way - the value within the Description attribute is the value (which presumably wants to be `@disabled`) - and the enum name is the constant name (presumably `Disabled`)